crawler/crawler.py

```diff
@@ -219,14 +229,8 @@ def main():
     else:
         raise NotImplementedError('Invalid crawlmode')
 
-    emitters = EmittersManager(urls=args.url,
```

Comment:
I liked this one better, when crawlers and emitters were separate, it was cleaner :(